test/unit/manipulation.js

```diff
@@ -2452,6 +2452,81 @@ test( "Validate creation of multiple quantities of certain elements (#13818)", 4
 	});
 });
 
+test( "Make sure tr element will be appended to tbody element of table when it present", function() {
```

Comment:
You can either say "when it is present" or just "when present" for the descriptions on these.